### DISCUSSION TITLE
[QA] Home

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#260] [Q&A]
Home QA 해주세욤!!

**Comment by Hyevvy** [comment]
[Card] 비로그인이 하트를 누르면 로그인 해달라는 alert를 띄워주는건 어떨까요? 누르니까 갑자기 상세페이지로 이동하네용 !!

**Reply by NamgyungKim** [reply]
넵 기능안하고 카드만눌리도록 했는데 로그인하겠는지 물어봐도 좋을것 같네요~~

**Reply by Hyevvy** [reply]
히힛 남경님 체고

**Comment by Hyevvy** [comment]
아니 이런.. notyet.. 흠흠.. ㅋㅋ
<img width="337" alt="스크린샷 2022-06-21 오후 6 28 42" src="https://user-images.githubusercontent.com/72402747/174766827-d96b9a36-074f-4734-bbf9-2b2a01565482.png">

**Reply by Hyevvy** [reply]
ㅋㅋㅋㅋㅋ 이거 제가 value말고 label로 드릴게용 ㅋㅋ

**Reply by Hyevvy** [reply]
오잉 근데 6개월 이상은 잘 뜨네요 ??
